tests/Makefile

```diff
@@ -0,0 +1,16 @@
+# Just a Makefile for automatic testing
+.PHONY: all
+
+all: clean init test
+
+clean:
+	rm -rf .jenkinsfile-runner-test-framework
+	rm -rf cwp.jar
+
+init:
+	git clone --depth 1 https://github.com/jenkinsci/jenkinsfile-runner-test-framework .jenkinsfile-runner-test-framework && cd .jenkinsfile-runner-test-framework && git checkout 0ed9ea7a43161ad97764db566d52412dd2dab621
```

Comment:
Beware of the `--depth 1`. Since we're forcing a commit, it might be not available if the commit history keeps growing.

Reply:
Yes, I have removed it.

Reply:
You've faster than me! :smile: